### PR TITLE
Task manifest with dynamic metrics

### PR DESCRIPTION
**Pull request by marcin-krolik**
Summary of changes:
- Updates task manifest to query for new dynamic metrics

How to verify it:
- execute example task

Testing done:
- manual

@intelsdi-x/plugin-maintainers

### CODE REVIEW

**Comment by marcin-krolik** [comment]
PR #34 should be merged before this one

File: scripts/config/k8s-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+metadata: 
+    name: psutil-deployment
+spec:
+    replicas: 1
+    template:
+        metadata:
+          labels:
+            app: psutil-large-test
+        spec:
+            - name: psutil-large-test
```
Comment (kindermoumoute) [inline scripts/config/k8s-deployment.yml]: There is a missing line here, it should be:
```
        spec:
            containers:
            - name: psutil-large-test
```

**Comment by kindermoumoute** [comment]
The issue is due to the fact that your container doesn't have entrypoint. In the Snap container the entrypoint is running Snap. Then tests are done with `kubectl exec`. In your example you run Snap within your large test script. So there are 2 possible solutions:
* Writing an entrypoint that will run Snap.
* Using a hacky entrypoint:
```
              ...
              imagePullPolicy: "IfNotPresent"
              command:
              - sleep
              - "120"
```

**Comment by marcin-krolik** [comment]
@kindermoumoute fixed `k8s` deployment file. I tested it manually on minikube.

**Comment by marcin-krolik** [comment]
Large test will fail until PR [metric too ambiguous](https://github.com/intelsdi-x/snap/pull/1225) is merged

**Comment by kindermoumoute** [comment]
You need to update links https://github.com/intelsdi-x/snap-plugin-collector-psutil/blob/master/scripts/test/large.py#L34-L35
to
```
http://snap.ci.snap-telemetry.io/snap/latest_build/linux/x86_64/snapd
http://snap.ci.snap-telemetry.io/snap/latest_build/linux/x86_64/snapctl
```

With those links updated, test took a while (5-10 minutes), but it worked.

**Pull request merged**